README.md

```diff
@@ -25,9 +25,9 @@ Periodic AWS E2E Virtualization Tests in OpenShift 4.17
 [![VM tests](https://prow.ci.openshift.org/badge.svg?jobs=periodic-ci-openshift-oadp-operator-master-4.17-e2e-test-kubevirt-aws-periodic)](https://prow.ci.openshift.org/job-history/gs/origin-ci-test/logs/periodic-ci-openshift-oadp-operator-master-4.17-e2e-test-kubevirt-aws-periodic)
 
 OADP repositories images job
-| OADP | OpenShift Velero plugin | Velero | Velero plugin for AWS | Velero plugin for GCP | Velero plugin for Microsoft Azure | Non Admin |
+| OADP | OpenShift Velero plugin | Velero | Velero plugin for AWS | Velero plugin for Legacy AWS | Velero plugin for GCP | Velero plugin for Microsoft Azure | Non Admin |
```

Comment:
formatting broke here 
![Screenshot from 2024-10-22 11-37-38](https://github.com/user-attachments/assets/c58d214f-afd8-4bcb-8b2d-fa45f24def88)

probably need one more column in line 29 (adding `---------- |` at the end)

Reply:
@mateusoliveira43 Updated.